examples/widgets/animimg/lv_example_animimg.c

```diff
@@ -0,0 +1,23 @@
+#include "../../../lvgl.h"
```

Comment:
```suggestion
#include "../../lv_examples.h"
```

It should solve teh build issue. 

For consistency please rename `LV_USE_ANIM_IMG` to `LV_USE_ANIMIMG`.

Reply:
Yes, it works, but I notice that the other examples (such as img, calendar) using "include ../../../lvgl.h".

Reply:
The includes in other examples were updated in master. 
See https://github.com/lvgl/lvgl/blob/master/examples/widgets/arc/lv_example_arc_1.c#L1